lib/Database/Migration/Production/Migration_0009_book_references_markdown/Data/BookReferences.hs

```diff
@@ -0,0 +1,986 @@
+module Database.Migration.Production.Migration_0009_book_references_markdown.Data.BookReferences where
+
+import qualified Data.Bson as BSON
+
+--bookReferencesMD :: UTCTime -> [(BSON.Selector, BSON.Document, [BSON.UpdateOption])]
```

Comment:
Is this valid type of function? If so, please remove it as comment. If not, please remove it all.

Reply:
It is valid but requires additional imports... probably unnecessary as it is not in other migrations as well... Fixed by https://github.com/DataStewardshipWizard/dsw-server/pull/26/commits/cdc514c36e058cfce186feeb5e5b1602d5da2663